README.md

```diff
@@ -25,6 +25,15 @@ go get -u -tags=gles2 github.com/go-gl/glfw/v3.2/glfw
 Supported tags are `gles1`, `gles2`, `gles3` and `vulkan`.
 Note that required packages might differ from those listed above; consult your hardware's documentation.
 
+### External GLFW library
+
+If your host system has GLFW library already compiled and installed,
+you can link against it, instead of compiling and statically linking the embedded one.
+You do this by passing `-tags=glfw_external` flag to `go build` or `go install`,
```

Comment:
The pattern for Go build constraints is to use no character to separate words. For example:

```
purego
netgo
nethttpomithttp2
```

We should follow that pattern by naming this `glfwexternal` rather than `glfw_external`.